packages/rule-tester/src/RuleTester.ts

```diff
@@ -871,18 +885,6 @@ export class RuleTester extends TestFramework {
               `messageId '${message.messageId}' does not match expected messageId '${error.messageId}'.`,
             );
 
-            const unsubstitutedPlaceholders =
-              getUnsubstitutedMessagePlaceholders(
-                message.message,
-                rule.meta.messages[message.messageId],
-                error.data,
-              );
-
-            assert.ok(
-              unsubstitutedPlaceholders.length === 0,
-              `The reported message has ${unsubstitutedPlaceholders.length > 1 ? `unsubstituted placeholders: ${unsubstitutedPlaceholders.map(name => `'${name}'`).join(', ')}` : `an unsubstituted placeholder '${unsubstitutedPlaceholders[0]}'`}. Please provide the missing ${unsubstitutedPlaceholders.length > 1 ? 'values' : 'value'} via the 'data' property in the context.report() call.`,
-            );
-
```

Comment:
Where did the `assert.ok(unsubstitutedPlaceholders.length === 0, ...` assertion go? All I see is that it has been deleted, but not restored elsewhere..